experimental/python/codegen/codegen/packages.py

```diff
@@ -0,0 +1,91 @@
+import re
+from typing import Optional
+
+RESOURCE_NAMESPACE_OVERRIDE = {
+    "resources.Job": "jobs",
+    "resources.Pipeline": "pipelines",
+    "resources.JobPermission": "jobs",
+    "resources.PipelinePermission": "pipelines",
+}
+
+# All supported resource types
+RESOURCE_TYPES = [
+    "resources.Job",
+    "resources.Pipeline",
+]
+
+# Namespaces to load from OpenAPI spec.
+#
+# We can't load all types because of errors while loading some of them.
+LOADED_NAMESPACES = [
+    "compute",
```

Comment:
The compute namespace is for job clusters, right?

Reply:
It's for job clusters and pipeline cluster configuration